ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/string/Uuid.java

```diff
@@ -31,4 +33,20 @@ public class Uuid {
   public String uuid() {
     return java.util.UUID.randomUUID().toString();
   }
+
+  @Udf
+  public String toUuid(@UdfParameter final ByteBuffer bytes) {
+    if (bytes == null) {
+      return null;
+    }
+
+    if (bytes.getCapacity() != 16) {
```

Comment:
lets combine this if condition with the above so that we have 
`if (bytes == null || bytes.getCapacity() != 16)  { return null;}`